core-api/src/main/java/com/sequenceiq/cloudbreak/validation/DeprecatedValidator.java

```diff
@@ -0,0 +1,17 @@
+package com.sequenceiq.cloudbreak.validation;
+
+import javax.validation.ConstraintValidator;
+import javax.validation.ConstraintValidatorContext;
+
+public class DeprecatedValidator implements ConstraintValidator<ValidDeprecated, String> {
+
+    @Override
+    public boolean isValid(String value, ConstraintValidatorContext context) {
+        if (value == null || value.isEmpty()) {
+            return true;
+        }
```

Comment:
I prefer having an `else` branch. Makes the code indentation more obvious about the exit point of the method